### ISSUE TITLE
Chop Bot 3000 not always offering the Runner to draw cards

### ISSUE THREAD

**Issue by francescopellegrini** [mtgred/netrunner#6679]
[In the current implementation](https://github.com/mtgred/netrunner/blob/8b77fe59ced47dd312d12d106ead134d6bbdc083/src/clj/game/cards/hardware.clj#L364) the Runner cannot choose to draw a card when the Stack is empty, even if this action could be replaced by other effects.

I think other cards do not apply this restriction and that we should align Chop Bot 3000 implementation.

**Comment by NBKelly** [comment]
agreed - chop bot should always be usable so long as there is an installed card to trash (which is the state change for the ability).
